rfc/0008-builtin-blocktypes/README.md

```diff
@@ -0,0 +1,212 @@
+<!--
+SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+
+SPDX-License-Identifier: AGPL-3.0-only
+-->
+
+# RFC 0008: Builtin blocktypes
+
+|             |                      |
+|-------------|----------------------|
+| Feature Tag | `builtin-blocktypes` |
+| Status      | `DISCUSSION`              | <!-- Possible values: DRAFT, DISCUSSION, ACCEPTED, REJECTED -->
+| Responsible | `@felix-oq`          |
```

Comment:
We should probably change the responsible person.

Reply:
Done in the latest commit, I think then we can go like this, right? If we need to iterate on this I'd rather create a new PR.